libs/types/src/system_settings.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from 'zod';
+import { Id, Iso8601Timestamp } from './generic';
+
+/**
+ * System settings as used by frontends. Several database fields are hidden from the app
+ * because they are omitted in this model; see schema.sql.
+ */
+export interface SystemSettings {
+  pollWorkerCardPinsEnabled: boolean;
```

Comment:
Totally feel free to ignore this, especially since I put this variable name in the original ticket 😅, but I know we often prefix boolean variable names with `is` or `are` for consistency/readability, so this could be `arePollWorkerCardPinsEnabled`

Reply:
https://github.com/votingworks/vxsuite/pull/3147/commits/1e30a9e3822ec9c80008fbe7a57366d0ed5da3a6